ext/opentelemetry-ext-prometheus/src/opentelemetry/ext/prometheus/__init__.py

```diff
@@ -0,0 +1,157 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Prometheus Metrics Exporter for OpenTelemetry."""
+
+import logging
+import re
+from typing import Sequence
+
+from prometheus_client import start_http_server
+from prometheus_client.core import (
+    REGISTRY,
+    CollectorRegistry,
+    CounterMetricFamily,
+    GaugeMetricFamily,
+    UnknownMetricFamily,
+)
+
+from opentelemetry.metrics import Counter, Gauge, Measure, Metric
+from opentelemetry.sdk.metrics.export import (
+    MetricRecord,
+    MetricsExporter,
+    MetricsExportResult,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class PrometheusMetricsExporter(MetricsExporter):
+    """Prometheus metric exporter for OpenTelemetry.
+
+    Args:
+        port: The Prometheus port to be used.
+        address: The Prometheus address to be used.
+        prefix: single-word application prefix relevant to the domain the metric belongs to.
+    """
+
+    def __init__(self, port: int = 8000, address: str = "", prefix: str = ""):
+        self._port = port
+        self._address = address
+        self._collector = CustomCollector(prefix)
+
+        start_http_server(port=self._port, addr=str(self._address))
+        REGISTRY.register(self._collector)
+
+    def export(
+        self, metric_records: Sequence[MetricRecord]
+    ) -> MetricsExportResult:
+        self._collector.add_metrics_data(metric_records)
+        return MetricsExportResult.SUCCESS
+
+    def shutdown(self) -> None:
+        REGISTRY.unregister(self._collector)
+
+
+class CustomCollector:
+    """ CustomCollector represents the Prometheus Collector object
+        https://github.com/prometheus/client_python#custom-collectors
+    """
+
+    def __init__(self, prefix: str = ""):
+        self._prefix = prefix
+        self._metrics_to_export = []
+
+    def add_metrics_data(self, metric_records: Sequence[MetricRecord]):
+        self._metrics_to_export.append(metric_records)
+
+    def collect(self):
+        """Collect fetches the metrics from OpenTelemetry
+        and delivers them as Prometheus Metrics.
+        Collect is invoked every time a prometheus.Gatherer is run
+        for example when the HTTP endpoint is invoked by Prometheus.
+        """
+
+        for metric_batch in self._metrics_to_export:
+            for metric_record in metric_batch:
+                prometheus_metric = self._translate_to_prometheus(
+                    metric_record
+                )
+                if prometheus_metric:
+                    yield prometheus_metric
+            self._metrics_to_export.remove(metric_batch)
+
+    def _translate_to_prometheus(self, metric_record: MetricRecord):
+        prometheus_metric = None
+        label_values = []
+        label_keys = []
+        for label in metric_record.label_set.labels:
+            for index, label_tuple_value in enumerate(label):
```

Comment:
Instead of calling `enumerate` and then checking on `index` you can just use a slice and step to get only the odd elements of `label`:

```python
for element in ["a", "b", "c", "d", "e"][1::2]:
    print(element)
```
```
b
d
```